### PR TITLE
chore: clean up dependencies

### PR DESCRIPTION
**Pull request by mantariksh**
This PR does the following:
1. Pins [dedent-js](https://www.npmjs.com/package/dedent-js) to v1.0.1, since it was last published 5 years ago and seems to be only maintained by 1 person.
2. Pins [sortablejs](https://www.npmjs.com/package/sortablejs) to v 1.10.2, since that is the last version which supports IE11.
3. If dependencies were actually dev dependencies, moves them under dev dependencies.
4. Removes unused dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -101,16 +99,14 @@
...
     "connect-mongo": "^3.2.0",
     "convict": "^6.0.0",
     "convict-format-with-validator": "^6.0.0",
     "cookie-parser": "~1.4.0",
     "crypto-js": "^4.0.0",
     "css-toggle-switch": "^4.1.0",
     "csv-string": "^4.0.1",
-    "dedent-js": "^1.0.1",
-    "deep-diff": "^1.0.1",
+    "dedent-js": "~1.0.1",
```
Comment (shuli-ogp) [inline package.json:109]: Just to confirm if this should not be exact version instead, based on the reasoning that we are using a package last published 5 years ago and don't want to allow any updates at all without reviewing closely

```suggestion
    "dedent-js": "1.0.1",
```

**Review by shuli-ogp — APPROVED** [review]
Just one clarification, lgtm otherwise

File: package.json (hunk 1 above)
Comment (mantariksh) [reply package.json:109]: my logic is that if someone decides they want to patch any bugs in the library, we should be able to apply it. wdyt?

Comment (shuli-ogp) [reply package.json:109]: sounds good!

**Pull request merged**